internal/stack/serverless.go

```diff
@@ -242,6 +247,11 @@ func (sp *serverlessProvider) BootUp(options Options) error {
 		return fmt.Errorf("serverless project type not supported: %s", settings.Type)
 	}
 
+	logstashEnabled := false
+	if options.Profile.Config("stack.logstash_enabled", "false") == "true" {
+		logstashEnabled = true
+	}
```

Comment:
Probably, this could be moved to `getProjectSettings()` function, so this value can be retrieved from `settings` variable.